config/prod.yml

```diff
@@ -1,8 +1,8 @@
 proxy:
   host: beebop.dide.ic.ac.uk
   ssl:
-    certificate: VAULT:secret/beebop/ssl:cert
-    key: VAULT:secret/beebop/ssl:key
+    certificate: VAULT:secret/beebop/ssl/production:cert
+    key: VAULT:secret/beebop/ssl/production:key
```

Comment:
We've added an additional level to keep ssl values for different servers (production and staging).